packages/rx_bloc_cli/mason_templates/bricks/feature_cicd_fastlane/__brick__/docs/continuous_delivery.md

```diff
@@ -146,6 +146,23 @@ The `deployment.yaml` contains necessary details used for deploying the app.
 In case of deploying the apps to the respective stores manually using the downloaded artifacts, please check the `Local distribution` section below.
 
 {{/cicd_codemagic}}
+{{#cicd_gitlab}}
+### Gitlab pipeline
+
+If created with the `--cicd=gitlab` flag, the generated project contains a workflow configuration file `.gitlab-ci.yml` at the root level. This file sets up the environment and runs the tests in your project. It is executed on every push or merge request in your GitLab repository.
+
+If you haven't created an access token for your distribution repository in GitLab, follow [this guide][clone_gitlab_repo_with_access_token] to do so, and keep the token handy.
+
+Within the `{project_root}/.gitlab-ci.yml` file, the default configuration sets up the environment to use a specific Flutter version. To change that configuration, update the `FLUTTER_VERSION` variable. You can also configure `MIN_COVERAGE` to set the minimum test code coverage threshold.
+
+To trigger the pipeline, push a change to your repository or create a merge request.
```

Comment:
This is already mentioned above

Reply:
I have removed it on the other place, so it will be more readable when add more options.